utils/appliance/__init__.py

```diff
@@ -378,6 +378,7 @@ def _configure_5_3(self, log_callback=None):
     def _configure_upstream(self, log_callback=None):
         self.wait_for_evm_service(timeout=1200, log_callback=log_callback)
         self.deploy_merkyl(start=True, log_callback=log_callback)
+        self.add_ca_certs(log_callback=log_callback)
```

Comment:
Is there a way to generate the templates before merge so it could be tested wit PRT?

I think the person doing merging should notify you. I have no merging rights, so I am not gonna be that person.